src/components/Project.js

```diff
@@ -2,21 +2,24 @@ import React from 'react'
 import styled from 'styled-components/macro'
 import ProjectHeadline from './ProjectHeadline'
 import PropTypes from 'prop-types'
+import Tags from './Tags'
 
-export default function Project({ title, country, description, image }) {
+export default function Project({ data }) {
   return (
     <Card>
-      <Img alt='' src={image ?? 'https://source.unsplash.com/random'}></Img>
-      <ProjectHeadline title={title} country={country} />
-      <Description>{description}</Description>
+      <Img alt='' src={data.image ?? 'https://source.unsplash.com/random'}></Img>
+      <TagList>
+        {data.tags.map((tag, index) => (
+          <Tags content={tag} key={index} />
+        ))}
+      </TagList>
+      <ProjectHeadline title={data.title} country={data.country} />
+      <Description>{data.description}</Description>
     </Card>
   )
 }
 Project.propTypes = {
-  title: PropTypes.string.isRequired,
-  country: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
-  image: PropTypes.string,
+  data: PropTypes.object.isRequired,
```

Comment:
Since you combined all your previous props into one object, it is now harder to see what kind of parameters you hand into the function.

If you like you can specify the "shape" your object should have like this:

```
optionalObjectWithShape: PropTypes.shape({
    optionalProperty: PropTypes.string,
    requiredProperty: PropTypes.number.isRequired
  }),
```

That makes it easier for other people to understand what the function receives as input.